packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -68,17 +99,25 @@ class FederationRuntimePlugin {
       });
     }
 
+    const embedRuntimeLines = embedRuntime
```

Comment:
dont have patch entry import federation runtime since it comes from require federation already in runtime module